airflow/www/templates/airflow/dags.html

```diff
@@ -125,7 +125,15 @@ <h2>DAGs</h2>
                   </div>
                 </td>
                 {# Column 3: Dag Owners #}
-                <td>{{ dag.owners }}</td>
+                <td>
+                  {% for owner in dag.owners.split(",") %}
+                  <a class="label label-default"
+                     href="?search={{ owner | trim }}"
+                     style="margin: 3px;">
```

Comment:
The tiniest adjustment will keep the left side of this left-aligned column just right:
```suggestion
                     style="margin: 3px 6px 3px 0;">
```